config.toml

```diff
@@ -18,7 +18,7 @@ githuburl = "https://github.com/spiffe/spiffe.io/tree/master"
 
 [params.home]
 acronym     = "**S**ecure **P**roduction **I**dentity **F**ramework **f**or **E**veryone"
-description = "Inspired by the production infrastructure of Google and others, SPIFFE is a set of open-source standards for securely identifying software systems in dynamic and heterogeneous environments. "
+description = "Industry best practice authentication for cloud-native services. "
```

Comment:
How about:

```suggestion
description = "Production identity control plane for distributed systems. "
```

Reply:
I'm open to changes here, and I like "identity control plane" as a description generally but I'm not comfortable that a new user to SPIRE would intuitively understand what an "identity control plane" is (and thus why they would want one). This might be something to user test or UX test before changing.

Reply:
There is not much that a new user will understand intuitively but at least they are in the right place. Here lies a unique opportunity to create a clear story, define a category blueprint, and shape the thinking of new users. 

The Benefits and use cases described below tie the concept together by giving them what an identity control plane is and why the would want one. 

New users will have arrived at this page knowing at least something. In the worst case, every single new user has some background knowledge of software or systems. They will know what a control plane is.

Reply:
Update: From the discussion below it sounds like this resonates. Let's try it :)